sys-power/powerstat/powerstat-0.02.11.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Laptop power measuring tool"
+HOMEPAGE="https://launchpad.net/ubuntu/+source/${PN} https://github.com/ColinIanKing/${PN}"
+SRC_URI="http://kernel.ubuntu.com/~cking/tarballs/${PN}/${P}.tar.gz
+	https://github.com/ColinIanKing/${PN}/archive/V${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+src_compile() {
+	emake CC=$(tc-getCC)
```

Comment:
just to double-check: have you checked that the Makefile respects `CFLAGS` and the lot?